platform/debian/do_debian_package.sh

```diff
@@ -38,20 +38,42 @@ if command_exists "${COMMAND}"; then
     {
         echo "Section: graphics"
         echo "Priority: optional"
-        echo "Depends: libsdl2-2.0-0"
+        echo "Depends: libsdl2-2.0-0, libc6 (>= 2.2.1)"
         echo "Architecture: ${ARCH}"
         echo "Version: ${VERSION}"
         echo "Installed-Size: $(du -ks "${INSTALL_DIR}/debian/usr/" | cut -f 1)"
 
-        echo "Package: KOReader"
-        echo "Maintainer: KOReader team"
+        echo "Package: koreader"
+        echo "Maintainer: KOReader team <dummy@koreader.rocks>"
```

Comment:
Are you sure it's not better just to let that error be?

Reply:
Nope, you're right :+1: